File: .net & mono/Weed3Demo/demo_plus/storeProcedure/user_get.cs
using Noear.Weed;
using System.Data;

namespace Weed3Demo.demo_ex {
    public class user_get : DbStoredProcedure {
        public user_get() : base(DbConfig.test) {
            call("user_get");
            set("_userID", () => userID);
        }

        public long userID;
    }
}

File: java/Weed3Demo/src/main/java/weed3demo/demo_plus/storeProcedure/user_get.java
package weed3demo.demo_plus.storeProcedure;

import org.noear.weed.DbStoredProcedure;
import weed3demo.config.DbConfig;

/**
 * Created by noear on 2017/7/22.
 */
public class user_get extends DbStoredProcedure {
    public user_get() {
        super(DbConfig.test);

        call("user_get");
        set("_userID", () -> userID);
    }

    public long userID;
}
